qiskit_nature/drivers/second_quantization/pyscfd/pyscfdriver.py

```diff
@@ -477,7 +477,7 @@ def _check_molecule_format(val: str) -> Union[str, List[str]]:
 
         # An xyz format has 4 parts in each atom, if not then do zmatrix convert
         # Allows dummy atoms, using symbol 'X' in zmatrix format for coord computation to xyz
-        parts = [x.strip() for x in atoms[0].split(" ")]
+        parts = [x.strip() for x in atoms[0].split()]
```

Comment:
Was lint or something complaining about explicitly providing the value that is the default. I guess its fine, but presumably either is ok.

Reply:
Actually no. But I found some weeks ago that this was causing problems because I had loading the coordinates from a file where multiple spaces were used in order to make the numbers aligned and more readable. So I had been using this in my local fork for quite a while and just noticed that this was not in the original source.